### ISSUE TITLE
In multshops  the default category of a each product is not saved when the same product is saved in multishops 

### ISSUE THREAD

**Issue by sextoysunivers1964** [PrestaShop/PrestaShop#27468]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

In multshops  the default category of a each product is not saved when the same product is saved in multishops.
Seems that products in the other shops don't save the default category as shown in the video 👍 
https://watch.screencastify.com/v/HhiFkNPmBnOKAN9EcIPd
 

### Expected behavior

To save the selected default category for a product.

### Steps to reproduce

1) Switch to single shop mode - "Sextoysunivers Sex shop online" - (The one with the ID shop = 2)
2) Open product ID 136145 , set the category to "Massage-Cremes" , make sure to check the DEFAULT CATEGORY flag ( https://www.screencast.com/t/CuMinWOtEf ) and save the product.
3) Switch to "All shops"  mode
4) Open product ID 136145 again , you'll see the default category flag is only one for all shops. Save the product again without editing the category.
5) Switch to single shop mode - "Sextoysunivers Sex shop online" - (The one with the ID shop = 2)
6) Open product ID 136145 again, see the "Massage-Cremes" default category flag is lost 

### PrestaShop version(s) where the bug happened

1.7.7.2

### PHP version(s) where the bug happened

7.3

**Comment by hibatallahAouadni** [comment]
Hello @sextoysunivers1964 

Your issue seems related to this one #22610
Could you please check and feedback.

Thanks!

**Comment by sextoysunivers1964** [comment]
Hello hibatallahAouadni,

I've checked that issue before to contact you.

It doesn't seem to be related cause in my case all the categories are there. 

Thank you !!

Best ragards,
Juan Carlos

**Comment by khouloudbelguith** [comment]
Hi @sextoysunivers1964,

Could you please confirm these exact steps to reproduce the issue?

https://user-images.githubusercontent.com/16067358/151591895-b9c9596e-42c9-4b12-8907-87f1675225c4.mp4

Thanks!

**Comment by sextoysunivers1964** [comment]
Hi,

The only difference with you is that there is not a default category (home or cream) when you come back to one of the shops as you can see in the attachment. 
<img width="900" alt="No Defaut in one shop" src="https://user-images.githubusercontent.com/84627578/151598300-e1ae66ce-4fe2-4fb7-b4ab-85def8847ac6.PNG">

Best regards;
Juan Carlos

**Comment by khouloudbelguith** [comment]
Hello @sextoysunivers1964,

>The only difference with you is that there is not a default category (home or cream) when you come back to one of the shops as you can see in the attachment.

I tried again with PS1778, I did not reproduce the issue.
The Home category is selected as a default category in two shops with different groups.

https://user-images.githubusercontent.com/16067358/153572041-11b28b56-99ed-4367-9917-668e81145a03.mp4

Despite our efforts, we were not able to reproduce your issue with the provided information. It seems that your issue is not a PrestaShop core bug but most likely a server configuration or customization problem.

We will close this issue.

Thank you!
